src/mev_custom_flood/sender.py

```diff
@@ -17,48 +20,51 @@
                     datefmt='%H:%M:%S',
                     level=logging.DEBUG)
 
+# this is the last prefunded address
+SENDER = os.getenv("SENDER_PRIVATE_KEY", "17fdf89989597e8bcac6cdfcc001b6241c64cece2c358ffc818b72ca70f5e1ce")
+# this is the first prefunded address
+RECEIVER = os.getenv("RECEIVER_PUBLIC_KEY", "0x878705ba3f8Bc32FCf7F4CAa1A35E72AF65CF766")
+EL_URI = os.getenv("EL_RPC_URI", 'http://0.0.0.0:53913')
 
-def flood():
-    # this is the last prefunded address
-    sender = os.getenv("SENDER_PRIVATE_KEY", "17fdf89989597e8bcac6cdfcc001b6241c64cece2c358ffc818b72ca70f5e1ce")
-    # this is the first prefunded address
-    receiver = os.getenv("RECEIVER_PUBLIC_KEY", "0x878705ba3f8Bc32FCf7F4CAa1A35E72AF65CF766")
-    el_uri = os.getenv("EL_RPC_URI", 'http://0.0.0.0:53913')
-
-    logging.info(f"Using sender {sender} receiver {receiver} and el_uri {el_uri}")
-
-    w3 = Web3(Web3.HTTPProvider(el_uri))
+def send_transaction():
+    # Setting w3 as constant causes recursion exceeded error after ~500 transactions
```

Comment:
does it take care of `nonce`s if you recreate it?

Pari has a PR where he sets the nonce as well!